tools/update-translations.sh

```diff
@@ -16,8 +17,15 @@ echo $HEADER > $LANGUAGE_DEF_FILE
 # Filter definition
 filter="current"
 strings_file="strings.xml"
+base_url="http://translate.wordpress.org/projects/apps/android/dev"
+
 if [ -n "$BUILD_TYPE" ]; then 
-    filter=$BUILD_TYPE; 
+    base_url="https://translate.wordpress.com/projects/wporg/apps/android";
```

Comment:
This depends on the layout of the projects in the GlotPress instance. Here, we are switching to the .com GlotPress _only_ for the translation review feature. The .org GlotPress is still the main and official source of the translations. 
Some languages are then sync'ed to the .com instance because Alex's team is building some tooling to support reviewers, including the custom app builds that we're tweaking here. 

I hope this is a bit more clear now. If you have any other doubts, please feel free to ask.

Reply:
> This depends on the layout of the projects in the GlotPress instance

👍 

And thanks for all the extra context! 😄